opentelemetry-api/src/opentelemetry/_metrics/__init__.py

```diff
@@ -401,6 +421,15 @@ class NoOpMeter(Meter):
     def create_counter(self, name, unit="", description="") -> Counter:
         """Returns a no-op Counter."""
         super().create_counter(name, unit=unit, description=description)
+        if self._check_instrument_id(name, DefaultCounter, unit, description):
+            _logger.warning(
+                "An instrument with name %s, type %s, unit %s and "
+                "description %s has been created already.",
+                name,
+                DefaultCounter.__name__,
```

Comment:
Should it be the base class `Counter` here so the error is consistence? Or maybe just string "Counter" instread of the FQN

Reply:
Right, fixed.